operator/internal/controller/auto/update_controller.go

```diff
@@ -643,26 +644,38 @@ func (s streamReader[T]) Result() (result, error) {
 
 		s.l.Info("Update complete", "result", res)
 
-		s.obj.Status.StartTime = metav1.NewTime(res.GetSummary().StartTime.AsTime())
-		s.obj.Status.EndTime = metav1.NewTime(res.GetSummary().EndTime.AsTime())
 		if link := res.GetPermalink(); link != "" {
 			s.obj.Status.Permalink = link
 		}
-		s.obj.Status.Message = res.GetSummary().Message
 		s.u.progressing.Status = metav1.ConditionFalse
 		s.u.progressing.Reason = UpdateConditionReasonComplete
 		s.u.complete.Status = metav1.ConditionTrue
 		s.u.complete.Reason = UpdateConditionReasonUpdated
-		switch res.GetSummary().Result {
-		case string(apitype.StatusSucceeded):
+
```

Comment:
Rationale here is that "preview" doesn't return a Summary object, so the overall result is a tad simplified in that case.